presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestPrestoDatabaseMetaData.java

```diff
@@ -94,6 +108,67 @@ public void testPassEscapeInMetaDataQuery()
         assertContains(query, "_t' ESCAPE '", "Metadata query does not contain ESCAPE");
     }
 
+    @Test
+    public void testGetTypeInfo()
+            throws Exception
+    {
+        ImmutableSet<Integer> typesToCheck = ImmutableSet.of(Types.BIGINT, Types.BOOLEAN, Types.INTEGER, Types.DECIMAL, Types.VARCHAR, Types.TIMESTAMP, Types.DOUBLE);
```

Comment:
We don't need this list, since the switch already contains the same list. (it'd be easy to add a case to the switch but forget to add it here)